src/model/libro.js

```diff
@@ -0,0 +1,48 @@
+import {ConjuntoGeneros} from "./generos.js"
+
+/**
+ * @class Libro Representa un libro
+ * Encapsula la información del título, autor, descripción y gérenos
+ * que definen a un libro
+ */
+export class Libro{
+   #titulo
+   #autor
+   #descripcion
```

Comment:
Repetidamente os he dicho que debéis buscar el producto *mínimamente* viable, y por supuesto que se ajuste a la historia de usuario. En ningún lugar en #7 ni en #12 (que, por cierto, no plantea un problema) dice lo que comentas en db371ce9ac6338aa07adad9de12c0b84e8875c24, "para que se haga una idea". La definición de deuda técnica es incluir código que no se ha pedido en ningún lugar como "solución" a un problema que en realidad no existe. Por favor, no lo hagáis.

Reply:
> Repetidamente os he dicho que debéis buscar el producto _mínimamente_ viable, y por supuesto que se ajuste a la historia de usuario. En ningún lugar en #7 ni en #12 (que, por cierto, no plantea un problema) dice lo que comentas en [db371ce](https://github.com/fjgallardo00/Akinah/commit/db371ce9ac6338aa07adad9de12c0b84e8875c24), "para que se haga una idea". La definición de deuda técnica es incluir código que no se ha pedido en ningún lugar como "solución" a un problema que en realidad no existe. Por favor, no lo hagáis.

Aquí es donde dice @JJ que #12 no plantea un problema (#7 es la HU). Por eso me replanteé si el error estaba en la manera de expresarlo. Por el resto tiene razón y ya he aprendido que si hay un issue incorrecto o innecesario, se borra el código añadido por ese issue y se cierra en el commit